server/@types/manage-recalls-api/models/AddDocumentRequest.ts

```diff
@@ -16,6 +16,7 @@ export namespace AddDocumentRequest {
     PART_A_RECALL_REPORT = 'PART_A_RECALL_REPORT',
     PREVIOUS_CONVICTIONS_SHEET = 'PREVIOUS_CONVICTIONS_SHEET',
     PRE_SENTENCING_REPORT = 'PRE_SENTENCING_REPORT',
+    RECALL_NOTIFICATION = 'RECALL_NOTIFICATION',
```

Comment:
Is this the same thing. as `RECALL_NOTIFICATION_EMAIL`, below?

Reply:
no idea, this is what happened when I ran swagger-to-ts

Reply:
Oh no, it is not the same thing.  The 'Recall notification' is now being saved when it is generated